src/IssueViz.UnitTests/Commands/IssueVisualizationToolWindowCommandTests.cs

```diff
@@ -38,41 +39,91 @@ public class IssueVisualizationToolWindowCommandTests
         {
             package = new Mock<AsyncPackage>();
             commandService = new Mock<IMenuCommandService>();
+            monitorSelection = new Mock<IVsMonitorSelection>();
             logger = new Mock<ILogger>();
 
             ThreadHelper.SetCurrentThreadAsUIThread();
+
+            var uiContext = new Guid(IssueVisualization.Commands.Constants.UIContextGuid);
+            monitorSelection.Setup(x => x.GetCmdUIContextCookie(ref uiContext, out uiContextCookie)).Returns(VSConstants.S_OK);
         }
 
         [TestMethod]
         public void Ctor_ArgsCheck()
         {
-            Action act = () => new IssueVisualizationToolWindowCommand(null, commandService.Object, logger.Object);
+            Action act = () => new IssueVisualizationToolWindowCommand(null, commandService.Object, monitorSelection.Object, logger.Object);
             act.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("package");
 
-            act = () => new IssueVisualizationToolWindowCommand(package.Object, null, logger.Object);
+            act = () => new IssueVisualizationToolWindowCommand(package.Object, null, monitorSelection.Object, logger.Object);
             act.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("commandService");
 
-            act = () => new IssueVisualizationToolWindowCommand(package.Object, commandService.Object, null);
+            act = () => new IssueVisualizationToolWindowCommand(package.Object, commandService.Object, null, logger.Object);
+            act.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("monitorSelection");
+
+            act = () => new IssueVisualizationToolWindowCommand(package.Object, commandService.Object, monitorSelection.Object, null);
             act.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("logger");
         }
 
         [TestMethod]
         public void Ctor_CommandAddedToMenu()
         {
-            new IssueVisualizationToolWindowCommand(package.Object, commandService.Object, logger.Object);
+            new IssueVisualizationToolWindowCommand(package.Object, commandService.Object, monitorSelection.Object, logger.Object);
 
             commandService.Verify(x =>
                     x.AddCommand(It.Is((MenuCommand c) =>
                         c.CommandID.Guid == IssueVisualizationToolWindowCommand.CommandSet &&
                         c.CommandID.ID == IssueVisualizationToolWindowCommand.CommandId)),
                 Times.Once);
+
+            commandService.Verify(x =>
+                    x.AddCommand(It.Is((OleMenuCommand c) =>
+                        c.CommandID.Guid == IssueVisualizationToolWindowCommand.CommandSet &&
+                        c.CommandID.ID == IssueVisualizationToolWindowCommand.ErrorListCommandId)),
+                Times.Once);
+
+            commandService.VerifyNoOtherCalls();
+        }
+
+        [TestMethod]
+        public void QueryStatus_FailsToQuery_NoException()
```

Comment:
`Fails` doesn't necessarily suggest an exception was thrown.
There's an existing test in this class called `Execute_ExceptionInCreatingWindow_NoException` which is less ambiguous, although I'd be more explicit and say `QueryStatus_NonCriticalException_IsSuppressed`.